Hippo.Core/Migrations/SqlServer/20240514163033_FixHistoryRelationship.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Hippo.Core.Migrations.SqlServer
+{
+    public partial class FixHistoryRelationship : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_Histories_Orders_OrderId",
+                table: "Histories");
+
+            migrationBuilder.DropForeignKey(
+                name: "FK_Histories_Orders_OrderId1",
+                table: "Histories");
+
+            migrationBuilder.DropIndex(
+                name: "IX_Histories_OrderId1",
+                table: "Histories");
+
+            migrationBuilder.DropColumn(
+                name: "OrderId1",
+                table: "Histories");
+
+            migrationBuilder.AddForeignKey(
+                name: "FK_Histories_Orders_OrderId",
+                table: "Histories",
+                column: "OrderId",
+                principalTable: "Orders",
+                principalColumn: "Id");
```

Comment:
Why is it dropping this one and adding it back? Shouldn't it just drop the other and leave this one alone?

Reply:
Don't know, but it updated the db correctly.

CREATE TABLE [dbo].[Histories](
	[Id] [int] IDENTITY(1,1) NOT NULL,
	[ActedDate] [datetime2](7) NOT NULL,
	[ActedById] [int] NULL,
	[AdminAction] [bit] NOT NULL,
	[Action] [nvarchar](100) NULL,
	[Details] [nvarchar](max) NULL,
	[ClusterId] [int] NOT NULL,
	[Status] [nvarchar](50) NULL,
	[OrderId] [int] NULL,
 CONSTRAINT [PK_Histories] PRIMARY KEY CLUSTERED 
(
	[Id] ASC
)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON, OPTIMIZE_FOR_SEQUENTIAL_KEY = OFF) ON [PRIMARY]
) ON [PRIMARY] TEXTIMAGE_ON [PRIMARY]
GO

ALTER TABLE [dbo].[Histories] ADD  DEFAULT ((0)) FOR [ClusterId]
GO

ALTER TABLE [dbo].[Histories]  WITH CHECK ADD  CONSTRAINT [FK_Histories_Clusters_ClusterId] FOREIGN KEY([ClusterId])
REFERENCES [dbo].[Clusters] ([Id])
ON DELETE CASCADE
GO

ALTER TABLE [dbo].[Histories] CHECK CONSTRAINT [FK_Histories_Clusters_ClusterId]
GO

ALTER TABLE [dbo].[Histories]  WITH CHECK ADD  CONSTRAINT [FK_Histories_Orders_OrderId] FOREIGN KEY([OrderId])
REFERENCES [dbo].[Orders] ([Id])
GO

ALTER TABLE [dbo].[Histories] CHECK CONSTRAINT [FK_Histories_Orders_OrderId]
GO

ALTER TABLE [dbo].[Histories]  WITH CHECK ADD  CONSTRAINT [FK_Histories_Users_ActedById] FOREIGN KEY([ActedById])
REFERENCES [dbo].[Users] ([Id])
ON DELETE CASCADE
GO

ALTER TABLE [dbo].[Histories] CHECK CONSTRAINT [FK_Histories_Users_ActedById]
GO